common/changes/@microsoft/rush/dotenv_2025-03-10-22-13.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add support for setting environment variables via `<repo-root>/.env` and `~/.rush-user/.env` files.",
```

Comment:
> ## Impacted documentation
>
> This should be documented.

@iclanton could you add a note to the top of https://rushjs.io/pages/configs/environment_vars/ ? That is probably the most discoverable place